orca-core/src/main/java/com/netflix/spinnaker/orca/config/UserConfiguredUrlRestrictions.java

```diff
@@ -123,6 +140,13 @@ public URI validateURI(String url) throws IllegalArgumentException {
         }
       }
 
+      for (String ip : rejectedIps) {
+        IpAddressMatcher ipMatcher = new IpAddressMatcher(ip);
+        if (ipMatcher.matches(u.getHost())) {
```

Comment:
It looks like you're passing the host name to `matches` here; is this function expecting the hostname or the IP address?

Reply:
The field name says address, but it only works with the hostname. Within the ```matches``` call, it calls InetAddress.getByName(address);

Reply:
Ah, I see---that's confusing, but it does look like the `parseAddress` actually converts a hostname to an address.  But now in the new commit you've reverted to passing an IP address...so is the new commit incorrect?

Reply:
Sorry I added that by accident. I must've tested wrong before because I can now confirm that the method works with either a hostname or an address. I will add a new commit to change it to the way it was before.

Reply:
Fixed